apps/create-fuels-counter-guide/playwright.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './test/ui',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
```

Comment:
Should `workers` ever be `undefined` ?

Reply:
This is [default playwright config](https://playwright.dev/docs/test-configuration#basic-configuration) tbf. I think undefined means it takes the default value (half the number of CPU cores).